EstateAdministrationUI/Areas/Estate/Controllers/HomeController.cs

```diff
@@ -311,52 +394,25 @@ public async Task<IActionResult> GetEstate(CancellationToken cancellationToken)
             return result;
         }
 
-        #endregion
-    }
-
-    /// <summary>
-    /// 
-    /// </summary>
-    /// <typeparam name="T"></typeparam>
-    public class DataTablesResult<T>
-    {
-        #region Properties
-
-        /// <summary>
-        /// Gets or sets the data.
-        /// </summary>
-        /// <value>
-        /// The data.
-        /// </value>
-        [JsonProperty("data")]
-        public IEnumerable<T> Data { get; set; }
-
         /// <summary>
-        /// Gets or sets the draw.
+        /// Validates the model.
         /// </summary>
-        /// <value>
-        /// The draw.
-        /// </value>
-        [JsonProperty("draw")]
-        public Int32 Draw { get; set; }
-
-        /// <summary>
-        /// Gets or sets the records filtered.
-        /// </summary>
-        /// <value>
-        /// The records filtered.
-        /// </value>
-        [JsonProperty("recordsFiltered")]
-        public Int32 RecordsFiltered { get; set; }
+        /// <param name="viewModel">The view model.</param>
+        /// <returns></returns>
+        private Boolean ValidateModel(CreateMerchantViewModel viewModel)
+        {
+            return this.ModelState.IsValid;
+        }
 
         /// <summary>
-        /// Gets or sets the records total.
+        /// Validates the model.
         /// </summary>
-        /// <value>
-        /// The records total.
-        /// </value>
-        [JsonProperty("recordsTotal")]
-        public Int32 RecordsTotal { get; set; }
+        /// <param name="viewModel">The view model.</param>
+        /// <returns></returns>
+        private Boolean ValidateModel(MakeMerchantDepositViewModel viewModel)
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'viewModel'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5745005)